demo/csv/CsvDemo.php

```
@@ -0,0 +1,141 @@
+        #[Input] int $age,
+        #[Input] string $role,
+        AgeGroup $ageGroup      // Injected service for age grouping
+    ): void
+    {
+        $ageGroup->addAge($age);
+        echo "dump1: Name: $name Email: $email Age: $age Role: $role\n";
+    }
+
+    public function dump2(
+        #[Input] UserInput $user, // Construct Input object with validation
+        #[Input] int $email,       // You can mix Input objects with primitive types
```

PHPMD (2.15.0)

Avoid unused parameters such as '$email'. (Unused Code Rules)

(UnusedFormalParameter)